gitpod-web/src/extension.ts

```diff
@@ -19,6 +20,8 @@ export async function activate(context: vscode.ExtensionContext) {
 		return;
 	}
 
+	gitpodContext.logger.info(`Gitpod Web ${meta.commit || meta.version}`);
```

Comment:
you can access package.json like this `const packageJSON = context.extension.packageJSON;`

Reply:
Nice, exactly what I was looking for, thanks JP!